spec/models/manageiq/providers/vmware/infra_manager/provision_spec.rb

```diff
@@ -205,6 +205,20 @@
         end
       end
 
+      context "#dest_storage_profile" do
+        let(:storage_profile) { FactoryGirl.create(:storage_profile, :name => "Gold") }
+
+        it "returns nil if no placement_storage_profile is given" do
+          @vm_prov.options[:placement_storage_profile] = nil
+          expect(@vm_prov.dest_storage_profile).to be_nil
+        end
+
+        it "returns a storage profile" do
+          @vm_prov.options[:placement_storage_profile] = [storage_profile.id, storage_profile.name]
+          expect(@vm_prov.dest_storage_profile).to eq(storage_profile)
+        end
```

Comment:
Can you add a `api_version < 5.5` test as well?  It should yield the same result as `it "returns nil if no placement_storage_profile is given"`.  But seems like it would be good for longevity to make sure someone doesn't accidentally remove the api_version check in the method and end up with bad results.